### ISSUE TITLE
Chat sorts by node name instead of time in the new ui

### ISSUE THREAD

**Issue by thespillmonkey** [MeshAddicts/meshinfo#115]
Chat sorts by node name instead of time in the new ui. no sort option for it.

**Comment by kevinelliott** [comment]
@thespillmonkey You are right. Chat will sort by timestamp ... additionally, we will be introducing pagination with some sane defaults. I'm not sure about sorting here, but can consider it.
